build.gradle

```diff
@@ -10,6 +10,7 @@ buildscript {
 allprojects {
     repositories {
         mavenCentral()
```

Comment:
btw, we should change our repo to jcenter() if possible, as that is Android Studio's default repo now.